polkadot/packages/react-api/src/Api.tsx

```diff
@@ -131,8 +130,8 @@ async function loadOnReady (api: ApiPromise, endpoint: LinkOption | null, inject
   const ss58Format = settings.prefix === -1
     ? chainSS58
     : settings.prefix;
-  const tokenSymbol = properties.tokenSymbol.unwrapOr([formatBalance.getDefaults().unit, ...DEFAULT_AUX]);
-  const tokenDecimals = properties.tokenDecimals.unwrapOr([DEFAULT_DECIMALS]);
+  const tokenSymbol = registry.createType('Text', 'CPAY');
+  const tokenDecimals = [DEFAULT_DECIMALS];
```

Comment:
so `tokenSymbol` and `tokenDecimals` are all coming from `properties` prop, is there no way we can do it without changing the `react-api` package directly?